frontend/src/views/FuelCodes/AddFuelCode/AddFuelCode.jsx

```diff
@@ -208,6 +240,37 @@ const AddFuelCodeBase = () => {
     },
     [saveRow]
   )
+  useEffect(() => {
```

Comment:
This is a lot of new functionality going into a ticket about writing tests, is this a bug fix?

Reply:
Yes, this is a bug fix. This functionality was initially requested via https://github.com/orgs/bcgov/projects/137/views/1?filterQuery=fuel+code&pane=issue&itemId=55073882&issue=bcgov%7Clcfs%7C433 but during ag-grid editor refactoring this was missed out. Hence adding this functionality back.

Reply:
raised the bug for this after confirming from Lindsy.
https://github.com/bcgov/lcfs/issues/1137